<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientInfoForm_Chat</class>
 <widget class="QWidget" name="ClientInfoForm_Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextBrowser" name="chatDisplay"/>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>메시지 입력</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_exit">
       <property name="text">
        <string>나가기</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_fileClient">
       <property name="text">
        <string>파일 첨부 하기</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_client">
       <property name="text">
        <string>전송하기</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
